apps/dedreckon/dedreckon.app.js

```diff
@@ -0,0 +1,442 @@
+/* Ded Reckon */
+/* eslint-disable no-unused-vars */
+
+/* fmt library v0.1.3 */
+let fmt = {
```

Comment:
Yes, that's long-term plan. But for that to work, I need to figure out what the interface should be, first, because  any interface change will break multiple applications at that point. I'm working in that direction... Plus, I'll have to figure out how to do development with modules, it seemed tricky last time I tried.

Reply:
Sure, that makes sense - let me know if you want a hand with any of it in later changes :)